docs/book/addresses.rst

```diff
@@ -4,24 +4,49 @@
 Addresses
 =========
 
-Every address in Sylius is represented by *Address* model. Default structure has the following fields:
+Every address in Sylius is represented by the **Address** model.
+It has a few important fields:
 
-* firstname
-* lastname
-* street
-* city
-* postcode
-* reference to *Country*
-* reference to *Province* (optional)
-* createdAt
-* updatedAt
+* ``firstName``
+* ``lastName``
+* ``phoneNumber``
+* ``company``
+* ``countryCode``
+* ``provinceCode``
+* ``street``
+* ``city``
+* ``postcode``
 
-Final Thoughts
---------------
+.. note::
 
-...
+   The Address has a relation to a **Customer** - which is really useful during the :doc:`Checkout addressing step </book/checkout>`.
+
+How to create an Address programmatically?
+------------------------------------------
+
+In order to create a new address, use a factory. Then complete your address with required data.
+
+.. code-block:: php
+
+   /** @var AddressInterface $address */
+   $address = $this->container->get('sylius.factory.address')->createNew();
+
+   $address->setFirstName('Harry');
+   $address->setLastName('Potter');
+   $address->setCompany('Ministry of Magic');
+   $address->setCountryCode('UK');
+   $address->setProvinceCode('UKJ');
+   $address->setCity('Little Whinging');
+   $address->setStreet('4 Privet Drive');
+   $address->setPostcode('000001');
+
+   $this->container->get('sylius.repository.address')->add($address);
```

Comment:
This is incorrect, address should not be added to the repository "alone", it can be, but shouldn't. :D Example with adding it to an order is enough. (persistence of the order is out of scope of this docs)